esi_ui/api/esi_api.py

```diff
@@ -34,63 +34,38 @@ def esiclient(token):
 def node_list(request):
     connection = esiclient(token=request.user.token.id)
 
-    def node_list_full_info(connection):
```

Comment:
are these changes part of the refactor in the previous PR? if so, you may want to rebase